tfhe/src/integer/oprf.rs

```diff
@@ -0,0 +1,76 @@
+use super::{RadixCiphertext, ServerKey};
+use crate::core_crypto::commons::generators::DeterministicSeeder;
+use crate::core_crypto::prelude::ActivatedRandomGenerator;
+use rayon::iter::{IndexedParallelIterator, IntoParallelIterator, ParallelIterator};
+
+pub use concrete_csprng::seeders::{Seed, Seeder};
+
+impl ServerKey {
+    /// Generates a `num_block` blocks integer
+    /// taken uniformly in `[0, 2^random_bits_count[` using the given a seed
+    /// It can be useful to make server random generation deterministic
```

Comment:
maybe indicate it's an encrypted random generation

server encrypted random generation

or

server FHE random generation

Reply:
Added "The encryted value is oblivious to the server"

Reply:
I think you can use the from_blocks on SignedRadixInteger, so basically the same function but wrapping differently, for the data range it will be much more tricky, so I guess we could start with the full range for the signed integer as a baseline, I think there would need to be a sign bit to manage, maybe worth a discussion with Thomas

Reply:
wrong comment thread sorry